src/redemptions/redemption.service.ts

```diff
@@ -430,4 +462,7 @@ export class RedemptionService {
 
     return { attemptable: true, reason: '' };
   }
+
+  minorAgeMessage = (age: number): string =>
+    `The age for your KYC (${age} years old) is lower than the legal requirement of 18. Please give custodianship to your parent/guardian and have them KYC.`;
```

Comment:
I prefer that we write simple english. I can immedietly tell that an engineer wrote this message.